daal4py/sklearn/monkeypatch/pairwise.py

```diff
@@ -151,13 +158,18 @@ def daal_pairwise_distances(X, Y=None, metric="euclidean", n_jobs=1, **kwds):
 
         dtype = bool if metric in PAIRWISE_BOOLEAN_FUNCTIONS else None
 
+        if (dtype == bool and
+                (X.dtype != bool or (Y is not None and Y.dtype != bool))):
```

Comment:
I find it more readable if the operator ('and') is in the next line.
Indentation is weird.